components/log-viewer-webui/server/src/main.ts

```diff
@@ -1,35 +1,72 @@
-import process from "node:process";
+// Reference: https://github.com/fastify/demo/blob/main/src/server.ts
 
-import app from "./app.js";
-import {
-    ENV_TO_LOGGER_MAP,
-    parseEnvVars,
-} from "./utils/env.js";
+import closeWithGrace from "close-with-grace";
+import fastify from "fastify";
+import fp from "fastify-plugin";
 
+import serviceApp from "./fastify-v2/app.js";
+
+
+const DEFAULT_FASTIFY_CLOSE_GRACE_DELAY = 500;
+const DEFAULT_PORT = 3000;
+
+/**
+ * Do not use NODE_ENV to determine what logger (or any env related feature) to use
+ *
+ * @return Logger options for Fastify.
+ * @see {@link https://www.youtube.com/watch?v=HMM7GJC5E2o}
+ */
+const getLoggerOptions = () => {
+    // Only if the program is running in an interactive terminal.
+    if (process.stdout.isTTY) {
+        return {
+            level: "info",
+            transport: {
+                target: "pino-pretty",
+                options: {
+                    translateTime: "HH:MM:ss Z",
+                    ignore: "pid,hostname",
+                },
```

Comment:
any reason we don't want to use the default settings?

Reply:
I think these options will only run in dev environment, not in production. I think the idea is too hide some information to make the logs less verbose, and make the timestamp more readable. If you want can remove, but I dont think impacts production logs.

Reply:
For local debugging, I believe using local time, rather than UTC time, is more intuitive. At our current scale, i agree we can hide pid in addition to hostname.
```suggestion
                options: {
                    ignore: "pid,hostname",
                },
```